libs/langchain/langchain/document_transformers/beautiful_soup_transformer.py

```diff
@@ -134,12 +137,14 @@ async def atransform_documents(
         raise NotImplementedError
 
 
-def get_navigable_strings(element: Any) -> Iterator[str]:
-    from bs4 import NavigableString, Tag
+def get_navigable_strings(element: Any, remove_comments: bool) -> Iterator[str]:
```

Comment:
nit: since this is technically a public method could we avoid breaking changes and give remove_comments a default value?

Reply:
This is a public method we introduced only a few days ago, so probably fine to leave as is, but up to you

Reply:
Thanks for the fast review. Good remark. For me this method `get_navigable_strings` should not really be public (it seems only intended to be used deep inside this code, not generally ?).

Could we make it more "private" (not _really_ private in Python, but anyway), and bring inside the BeautifulSoupTransformer class? Maybe as a static method inside the class, like the other methods?

Then we could also do the import only once in the init, or re-use the `import bs4` command that is already present there?

If you think that is better, I could try to adapt the code in that direction (to a class_method inside the class).

My more fundamental question on "breaking interface" is the question if the "remove_comments" should be default _True_ , like in my current proposal (but I am open to the alternatives as well ...).

Reply:
id minorly prefer to make it default to False (to your point about backwards compat) and also to add a default here (seems easy)